client/src/components/Slider/Slider.js

```diff
@@ -0,0 +1,107 @@
+import imageFile1 from "../../Images/Fall-2020-1.png";
+import imageFile2 from "../../Images/Fall-2020-2.png";
+import imageFile3 from "../../Images/Spring-2020-1.jpeg";
+import imageFile4 from "../../Images/Spring-2020-2.jpeg";
+import imageFile5 from "../../Images/Trending-Now-1.jpeg";
+import imageFile6 from "../../Images/Trending-Now-2.jpeg";
+import imageFile7 from "../../Images/Classics-1.jpeg";
+import imageFile8 from "../../Images/Classics-2.jpeg";
```

Comment:
These images should be passed down as props.

Reply:
Should I make a file for this data and pass it from that file to Slider.js with props?

Reply:
I agree that you can make an export file that holds all the images. But you should import said file into the stories.js file and pass them down as props.

Reply:
Please refer to the comment directly below where the array of data is displayed.